### ISSUE TITLE
MW4 needs an additional "post meridian flip settling time" user settable parameter

### ISSUE THREAD

**Issue by tonk777** [mworion/MountWizzard4#321]
This is just an observation and a request for enhancement :)

Currently Mw4 supports under "Settings" -> "Mount", the parameter "Waiting time after mount finished slew"

I currently have this set at 5 seconds for general slew and get very good results. However after a meridian flip when the scope, filter wheel, focuser and camera are rapidly inverted in orientation my rig needs a much longer settling time - in the order of 2 minutes to reach stability. 

Currently to get good model results I have to run ordered East to West to minimise the merdian flip to just one flip, and I always have to prune out the first 5 or so points after the meridian flip (RMS in order of 40" for first point after flip decreasing to under 5" after 4 additional points)

So can we have an additional parameter ... "Settings" -> "Mount" -> "Waiting time after mount finished MERIDIAN FLIP"

**Comment by tonk777** [comment]
Hi - is this being accepted as a new feature and if so which release will have it? Asking for a friend too :)

**Comment by mworion** [comment]
Yes, it should be in the next release 3.2.

Implementation is already done and quickly tested at home. Are you willing to volunteer as beta tester? 

Michel


Von meinem iPad gesendet

**Comment by tonk777** [comment]
"Are you willing to volunteer as beta tester?"

Yes :) - and I have another tester lined up with a rig in Spain (mine is in Chile)

**Comment by mworion** [comment]
Hi Tonk,
there is a v3.2.0b0 version online. If you are on v3 stream anyway, please check show beta in the local installer and you will see the version for test. If not, please for betas as always: separate work dir und do a second install.
The explanation how it should be setup is here : [https://mworion.github.io/MountWizzard4/config/mount/settlingTime.html](url)
Michel

**Comment by Ruediger3** [comment]
Hi Michel,

thanks for the feature. I have update and I am a bit confused about it. 
Unfortunately the link given above is not working for me. 

Rüdiger

**Comment by mworion** [comment]
Link should work now

**Comment by Ruediger3** [comment]
Link works now and I have read it, but I have to admit, I am struggling with it. Maybe I need a calm minute to understand it. 
Rüdiger

**Comment by Ruediger3** [comment]
Hi Michel,
I need your help: how to I add extra time after a flip, without extending the time between the slews?
Rüdiger

**Comment by mworion** [comment]
Rüdiger,
Ideally the standard mount settle time is OK for normal operation and also for modeling, so you would have a settling time for your mount set (upper image, in my case I have 1s set). If you would like to have an extended time only after flip, you set the additional time to 0s without flip and for the addition time after flip to x seconds (parameters shown in the second image).
But it might be a good idea to add an example in the doc.
Hope this helps.
Michel

Von meinem iPad gesendet

**Comment by Ruediger3** [comment]
Hi Michel,
Thanks for the help. Slowly I think I understand the logic.
I think the logic is very confusing. Why not having simply three fields slew, settle and flip settle time? This is much more versatile, flexible and all parameters could be kept in one place. Also they are not mutually impacting. All parameters can be set independently and can be combined as needed. 
I really had trouble to understand it, because the labels could be interpreted also in different way. 
Rüdiger

**Comment by mworion** [comment]
Any help and ideas appreciated.
Reason why I put this into different locations:
Settling time is managed in mount itself and could be altered with many tools.
I cannot use this parameter to adjust flip etc as I would change the basic behavior of the mount for many use cases without the knowledge of the user. If the user kills MW4 an altered parameter would be in the system. 
I cannot add this feature after flip longer settle time to the system itself as I build a workaround as I cannot access the system for that.
So I tried to split this by settling time and a parameter which helps during modeling.
What is thinkable if you really need two waiting times as if the settling is done correct, you normally would set the addition wait time without flip to zero.
Michel

Von meinem iPad gesendet

**Comment by Ruediger3** [comment]
Hi Michel,
Many thanks for explaining. These are good reasons. For my point of view the mount settle time in the mount plus a separate wait time for flip would be sufficient. Reason: if the settle time in the mount is good for imaging, it is by far good enough for the model build. I don’t see any practical benefit for extending it during build.

but what I would suggest is not to distribute the parameters over the tabs. I would try to keep them side by side, since they logically belong together.
What is your opinion?

Rüdiger

**Comment by mworion** [comment]
OK, 
it makes sense to remove the non flip option. 
Still I struggle a little bit to put it on the same UI as there might rise the wrong expectation to enable a longer settle time for all use cases of the mount. I would like to keep them separate or as I had before to remove to set the 10micron mount settling time at  all as it was not present before anyway.

@tonk777 Any other thoughts?
Michel

**Comment by mworion** [comment]
Further Thougts:

I tried to consolidate it to one place. There is another obstacle as for the settling time the mount takes car and does persistence for wait time it's MW4 and it's config file. So far I do not persist any value which belongs to a parameter set inside the mount and I would like to keep it. So I have to poll these values and GUI interaction is different. You might have experienced it, that for mount persisted values there is the need of a popup, whereas in MW4 persisted you just could alter the values in place.

Michel

**Comment by mworion** [comment]
Version 3.2.0b1 (due to next days) will have the version I would like to keep. Any news about the function itself ?

**Comment by Ruediger3** [comment]
Hi Michel,
I have nothing to add. I only need additional time after slew. I think Tonk owes you a feedback.😉
Life I could not test due to clouds. 

Cheers 
Rüdiger

**Comment by tonk777** [comment]
Hi I'm still on holiday. Can't do much from a campervan!

Will test this out  when I get back at the weekend

Tony (Tonk)
________________________________
